src/components/Sidebar/LanguageDropdown.tsx

```
@@ -0,0 +1,117 @@
+
+  useOnClickOutside(() => setMenuOpen(false), selectRef);
+
+  const options = Object.entries(languageVersions)
+    .map(([lang, version]) => ({
+      label: lang as LanguageKey,
+      value: `${lang}-${version}`,
+      version,
+    }))
+    .toSorted((a, b) => a.label.localeCompare(b.label));
+
+  const queryParams = new URLSearchParams(typeof window !== `undefined` ? window.location.search : '');
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)